llama_index/evaluation/retrieval/evaluator.py

```diff
@@ -23,25 +24,42 @@ class RetrieverEvaluator(BaseRetrievalEvaluator):
     Args:
         metrics (List[BaseRetrievalMetric]): Sequence of metrics to evaluate
         retriever: Retriever to evaluate.
+        post_processor (Optional[BaseNodePostprocessor]): Post-processor to apply after retrieval.
+
 
     """
 
     retriever: BaseRetriever = Field(..., description="Retriever to evaluate")
+    post_processor: Optional[BaseNodePostprocessor] = Field(
+        default=None, description="Optional post-processor"
+    )
 
     def __init__(
         self,
         metrics: Sequence[BaseRetrievalMetric],
         retriever: BaseRetriever,
+        post_processor: Optional[BaseNodePostprocessor] = None,
```

Comment:
Can we make this a list of postprocessors? And change the kwarg name to match the query engine? I.e. `node_postprocessors`

Reply:
Thanks for good feedback. Will fix shortly.

Reply:
Fixed the typing and added a loop to apply the postprocesses.